isomorphe/transformations/default/warning.xsl

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Retourne systematiquement un warning, en plus du XML d'input.
+-->
+
```

Comment:
Nitpick, I cleaned up other xslts so let's make this one consistent now.
Only the top part changes :
```
<?xml version="1.0" encoding="UTF-8"?>
<!--
Retourne systematiquement un warning, en plus du XML d'input.
-->
<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
                xmlns:gmd="http://www.isotc211.org/2005/gmd">
```